README.runtime.md

```diff
@@ -79,6 +79,22 @@ Tag | Dockerfile
 
 You can retrieve a list of all available tags for dotnet/framework/runtime at https://mcr.microsoft.com/v2/dotnet/framework/runtime/tags/list.
 
+# Version Compatibility
+
+Version Tag | OS Version | Supported .NET Versions
+-- | -- | --
+`4.8` | _all supported_ | 4.8
```

Comment:
I wonder if the use of _all supported_ will be confusing and/or cause people to scroll between this table and the tags listing.  Would listing out the versions help?  I mainly wonder this because the definition of `all supported` is not uniform across the versions.

Version Tag | OS Version | Supported .NET Versions
-- | -- | --
`4.8` | `windowsservercore-2009`, `windowsservercore-2004`, `windowsservercore-1909`, `windowsservercore-1903`, `windowsservercore-ltsc2019`, `windowsservercore-ltsc2016` | 4.8
`4.7.2` | `windowsservercore-ltsc2019`, `windowsservercore-ltsc2016` | 4.7.2
`4.7.1` | `windowsservercore-ltsc2016` | 4.7.1
`4.7` | `windowsservercore-ltsc2016` | 4.7
`4.6.2` | `windowsservercore-ltsc2016` | 4.6.2
`3.5` | `windowsservercore-2009`, `windowsservercore-2004`, `windowsservercore-1909`, `windowsservercore-1903` | 4.8, 3.5, 3.0, 2.5
`3.5` | `windowsservercore-ltsc2019` | 4.7.2, 3.5, 3.0, 2.5
`3.5` | `windowsservercore-ltsc2016` | 4.6.2, 3.5, 3.0, 2.5

Reply:
I debated this myself.  Your reasoning makes sense.